### PR TITLE
feat: Update qualification metadata to v2

### PR DESCRIPTION
**Pull request by LucsT**
Update to qualification v2 data and updating the manifest.

@paultranvan   Please note that the categories (isp/phone) is not set anymore in this format. So i deleted so detection code.
This data are conform to the data that the cozy-drive service generate now.
Is it a correct behaviour ?

@doubleface  Please note and approuve the tag in manifest DOC_QUALIFICATION_V2

### CODE REVIEW

**Comment by LucsT** [comment]
New version using directly models available through cozy-client in cozy-konnectors-libs.

This works very well, but broke the standalone mode.

**Review by doubleface — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -86,10 +90,15 @@ module.exports = new BaseKonnector(async function fetch(fields) {
...
-      const categories = []
-      // some bills are related to isp and phone
-      if (isIsp) categories.push('isp')
-      if (isMobile) categories.push('phone')
+      let qualification = Qualification.getByLabel('telecom_invoice')
+      // some bills are related to isp and phone, try a better qualification
+      if (isIsp) {
+        qualification = Qualification.getByLabel('isp_invoice')
+      }
+      if (isMobile) {
```
Comment (paultranvan) [inline src/index.js:98]: Is it possible to have both `isIsp` and `isMobile`? If yes, that would fall in `phone_invoice`, while I assume it should be `telecom_invoice`?

Comment (LucsT) [reply src/index.js]: Switch to an exclusive precise label and switch back to telecom_invoice if we have multi lines (mobile and not) in the same bill.

Comment (LucsT) [reply src/index.js]: Changed

**Review by paultranvan — APPROVED** [review]

**Pull request merged**